crates/ty_python_semantic/src/module_resolver/resolver.rs

```diff
@@ -158,7 +160,7 @@ pub struct SearchPaths {
     /// The Python version for the search paths, if any.
     ///
     /// This is read from the `pyvenv.cfg` if present.
-    python_version: Option<PythonVersionWithSource>,
+    python_version_from_metadata: Option<PythonVersionWithSource>,
```

Comment:
What does `from_metadata` mean here? Is it that the information comes from the `ProjectMetadata`? Or is it something else? If it's something else, then I'd prefer to either use a different name or extend the documentation (which suggests that it is from `pyenv`, in which case I would name it `pyenv_python_version`)